lib/xendit_api/api/disbursement.rb

```diff
@@ -6,8 +6,8 @@ module Api
     class Disbursement < XenditApi::Api::Base
       PATH = '/disbursements'.freeze
 
-      def create(params)
-        response = client.post(PATH, params)
+      def create(params, headers = {})
+        response = client.post PATH, params, headers
```

Comment:
wow, you can do that without () ? 
we can do it like this ya?         response = client.post PATH, params, headers

Instead of this one         response = client.post(PATH, params, headers)

Reply:
yes, in Ruby we can call method without brackets. I'm not sure which way is recommended, but rubocop ok with it.